app/src/main/res/layout/main_fragment.xml

```diff
@@ -88,6 +88,16 @@
         app:layout_constraintStart_toStartOf="@id/start_guideline"
         app:layout_constraintTop_toBottomOf="@id/audio_call_button" />
 
+    <Button
+        android:id="@+id/clear_session_button"
+        android:layout_width="0dp"
+        android:layout_height="51dp"
+        android:layout_marginTop="8dp"
+        android:text="Clear Session"
```

Comment:
Extract this to `strings.xml`

Reply:
fixed - b28f19808b0641620ed986b8fa412dfcd2597739